packages/storybook-vue/src/stories/Textarea/TextareaBinding.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textarea } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textarea-with-form-binding" title="Vue.js Component/Textarea" component={Textarea} argTypes={argTypes} />
+
+# Textarea
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textarea },
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textarea v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textarea Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
+  `,
+  methods: { action: action("modelValue") },
+});
```

Comment:
The real-time binding doesn't seem to work for me, unfortunately. The next never updates, even though the action panel shows the change.

<img width="561" alt="Screenshot 2022-08-02 at 13 01 34" src="https://user-images.githubusercontent.com/30694/182359458-c19c69b9-ea58-487c-b0b9-ca53825381d1.png">

Reply:
Yes, that is correct, I tried to solve it by following this [solution](https://craigbaldwin.com/blog/updating-args-storybook-vue/) but it seems the binding will be not in sync with the `modelValue` field below in the controls.
Only the `modelValue` in the controls change the value
<img width="779" alt="Screenshot 2022-08-02 at 14 17 23" src="https://user-images.githubusercontent.com/53454241/182372573-06edce5c-9f13-4153-8c6c-83f6e2aa0deb.png">
e

Reply:
I suspect having it out of sync like this will be quite confusing to folks. How about we remove this story for now, and include it once we have a syncing solution?